src/Ldap.php

```diff
@@ -1283,6 +1283,57 @@ public function delete($dn, $recursively = false)
     }
 
     /**
+     * Delete single attributes from an LDAP-Node
+
+     * This method removes single attributes from a node identified by
```

Comment:
What syntax are you using here? I see a mix of docbook and markdown blocks which:

* Inconsistent
* Docbook is not a good choice
* Markdown blocks should be replaced with inline blocks "`foo`"
* Use bracket syntax for arrays